db_model.py

```diff
@@ -0,0 +1,81 @@
+ 
+from sqlalchemy import (
+    create_engine,
+    Column,
+    ForeignKey,
+    Integer,
+    String,
+    TIMESTAMP,
+    Boolean,
+    Enum,
+    ARRAY,
+    JSON,
+    FLOAT
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+import os
+
+Base = declarative_base()
+
+user = os.environ.get("POSTGRES_USER")
+password = os.environ.get("POSTGRES_PASSWORD")
+host = os.environ.get("POSTGRES_HOST")
+db = os.environ.get("POSTGRES_DB")
+
+engine = create_engine("postgresql://" + user + ":" + password + "@" + host + "/" + db)
+print('connected to sql')
+
+class Event(Base):
+    __tablename__ = "event"
+
+    def __str__(self):
+        return str(self.__class__) + ": " + str(self.__dict__)
+
+    id = Column(Integer, primary_key=True)
+    organization_id = Column(Integer, ForeignKey("organization.id"))
+    location_id = Column(Integer, nullable=True)
```

Comment:
Add foreign key

Reply:
not that important

Reply:
I copy from your code.

Reply:
https://github.com/hu-tao-supremacy/participant/blob/1d73155b2422509e499b6d3ab64ba6ec22ab449b/db_model.py#L33

Reply:
FIxed, just make sure to fix your code.